package.json

```diff
@@ -9,16 +9,17 @@
   "license": "MIT",
   "main": "dist/app.js",
   "scripts": {
-    "clean": "rm -rf dist",
+    "clean": "rimraf dist",
     "build": "yarn run clean && webpack-dashboard -- webpack --config configs/webpack.config.js --env.build --env.sourceMap",
     "dev": "webpack-dashboard -m -- webpack-dev-server --config configs/webpack.config.js --open",
     "dev2": "webpack-dashboard -- webpack-dev-server --config configs/webpack.config.js --open --env.awesome",
-    "lint": "tslint --project 'tsconfig.json'",
+    "lint": "tslint --project \"tsconfig.json\"",
     "tsc": "tsc -p . --noEmit",
     "tsc:watch": "tsc -p . --noEmit -w",
     "test": "jest --config jest.json",
     "test:update": "jest --config jest.json -u",
     "re": "rm -rf node_modules && yarn",
+    "rewin": "rd /s /q node_modules && yarn",
```

Comment:
By the way, speaking of the readme, shouldn't that show yarn, instead of npm in the command examples?